<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyChan</class>
 <widget class="QWidget" name="MyChan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridL" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="1,5,1">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item row="9" column="1">
      <widget class="QLineEdit" name="NewPassEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="20" column="1">
      <widget class="QPushButton" name="ConnectToMyChanBtn">
       <property name="text">
        <string>Connect to my channel</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="ConNewPassLabel">
       <property name="text">
        <string>Confirm new password channel:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="NewPassLabel">
       <property name="text">
        <string>New password channel:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="22" column="1">
      <widget class="QPushButton" name="DeleteMyChanBtn">
       <property name="text">
        <string>Delete my channel</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLineEdit" name="MaxCountUsersEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;My channel settings&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="MaxCountUsersLabel">
       <property name="text">
        <string>Max count users:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="16" column="1">
      <widget class="QPushButton" name="SetMCUBtn">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="ConNewPassEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="NameChannelEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="ChangeNameBtn">
       <property name="text">
        <string>Change</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="NameChannelLabel">
       <property name="text">
        <string>Name channel</string>
       </property>
      </widget>
     </item>
     <item row="23" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="12" column="1">
      <widget class="QPushButton" name="ChangePassBtn">
       <property name="text">
        <string>Change</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="OldPassLabel">
       <property name="text">
        <string>Old password channel:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="OldPassEdit"/>
     </item>
     <item row="21" column="1">
      <widget class="QPushButton" name="StateChanBtn">
       <property name="text">
        <string>Start channel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
